en/drafts/originals/space-place-gazetteers.md

```diff
@@ -25,122 +25,111 @@ doi: XX.XXXXX/phen0000
+Thus, in its briefest sense, a gazetteer is a dictionary or list of place names. However, a well-structured gazetteer reflects the fact that places are conceptual entities, not simply names or points on a map. For example, any given place may have had multiple names in numerous languages over the course of history, potentially involving conflicts about who has the power to enforce any of those names. The spatial extents, names, and feature types (settlements, buildings, nations, mountains, and so on) of places also frequently change over time. 
 
-Through this lesson, You will learn how to think about the concept of place, why gazetteers are useful for spatial history, how to use historical information to create a gazetteer, and how to enhance and share a gazetteer.
+This lesson teaches you how to leverage the power of digital gazetteers, which are essential resources for spatial history. Unlike maps, gazetteers can readily connect named spatial entities with one another and with their modern locations, and they make it easy to annotate any identified place with information about texts, events, people, or other places that have been associated with it. 
 
 ### Learning Outcomes
 
-This lesson will demonstrate how to build digital gazetteers, starting with simple spreadsheets and then building them into linked open data resources to share with other projects. Linked Data is structured data that can be interlinked with other data to undertake large queries. Linked Open Data is Linked Data that is released under an open license, meaning that it can be reused for other projects. You can find out more about Linked Open Data in [this *Programming Historian* Tutorial](https://programminghistorian.org/en/lessons/intro-to-linked-data).
+Throughout this lesson, you will learn how to think about the concept of place, why gazetteers are useful for spatial history, how to use historical information to create a gazetteer, and how to enhance and share a gazetteer.
 
-At the end of this lesson, you will be able to:
+This lesson will demonstrate how to build a digital gazetteer, starting with a simple spreadsheet that you can build into 'Linked Open Data' resources to communicate with other projects. [Linked Data](https://en.wikipedia.org/wiki/Linked_data) is structured data that can be interlinked with other data to undertake large queries. 'Linked Open Data' is Linked Data that is released under an open license, meaning that it can be reused for other projects. You can find out more about Linked Open Data in [this *Programming Historian* introduction to linked data](https://programminghistorian.org/en/lessons/intro-to-linked-data).
```

Comment:
I've capitalised Linked Data and Linked Open Data to stay consistent with the other lesson mentioned here, <https://programminghistorian.org/en/lessons/intro-to-linked-data>.

Reply:
This is a good change. Please do not put Linked Open Date in quotation marks. It should not appear that way.